lib/Installation/Partitioner/LibstorageNG/v4_3/ResizePage.pm

```diff
@@ -24,7 +24,7 @@ sub new {
 
 sub init {
     my $self = shift;
-    $self->{tb_size}        = $self->{app}->textbox({id => '"Y2Partitioner::Dialogs::BlkDeviceResize::CustomSizeWidget"'});
+    $self->{tb_size}        = $self->{app}->textbox({id => '"Y2Partitioner::Dialogs::PartitionSize::CustomSizeInput"'});
```

Comment:
Could you please run some test to confirm that we won't break other tests with this change.

Reply:
I've triggered https://openqa.suse.de/t5182003 which uses resizing methods.

Reply:
yes, that suprised me, I found this one for TW and it worked for me too.

Reply:
I guess this might be different for resizing and creating new partition. But I'm not sure why we need change resizing page for msdos, as we add partition there.

Reply:
So I reverted this, as it seems like we can just use the old implementation.